homeassistant/components/fitbit/const.py

```diff
@@ -64,198 +64,231 @@ class FitbitSensorEntityDescription(SensorEntityDescription, FitbitRequiredKeysM
         name="Activity Calories",
         unit_type="cal",
         icon="mdi:fire",
+        state_class="total_increasing",
```

Comment:
It's better to use the available constants to set the `state_class`, e.g. `state_class=SensorStateClass.TOTAL_INCREASING`.

Reply:
Ok Will fix that